### PR TITLE
Buffs Deforest Medical imports

### PR DESCRIPTION
**Pull request by YakumoChen**
## About The Pull Request

Adds a bunch of medical MOD modules and a few cyber augments to Deforest Medical's selection.
MOD stuff costs the same as mid-tier modules from Nakamura.
Augments cost a little more than organs on average.
Both need some company investment (the augs need more)

## How This Contributes To The Skyrat Roleplay Experience

Gonna be honest, as a doctor I've never had a reason to buy anything from Deforest, much less as any other job on the station.
The surgical tools will get me generally yelled at by admins if I'm not a doctor, and aren't really worth the company investment / money before they're available in the protolathe. The rest of Deforest's selection is pretty boring and I wouldn't say very worth the time and money spent. Maybe the upgraded medkit supplies, but chems generally outperform those anyway.

Anyway these add some proper incentives to invest into the company, and having the MODs here is good when the MOD company is already overloaded with top tier buys. MOD stuff is researched relatively late so there's plenty of incentive to buy these.
Aug stuff gives crew incentives to get augmented, although the selection I chose benefits med docs more than any other job.

## Proof of Testing

image

## Changelog

:cl:
balance: Deforest Medical imports now have a selection of Medical-themed MODsuit modules for sale, as well as some cyber augments.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm [hunk 1]
```diff
@@ -8,9 +8,16 @@
 #define PRICE_CHEM_CHEAP_LOWER 5.7
 #define PRICE_CHEM_CHEAP_HIGHER 4.8
 #define PRICE_CHEM_MEDIUM_LOWER 4.4
-#define PRICE_CHEM_MEDIUM_HIGHER 4 
+#define PRICE_CHEM_MEDIUM_HIGHER 4
 #define PRICE_CHEM_PREMIUM_LOWER 3.5
 #define PRICE_CHEM_PREMIUM_HIGHER 3
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm]: Is it normal that here the lower are higher than the higher, and that the ones you added are inverted (higher is higher than lower)?

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm (hunk 1 above)
Comment (YakumoChen) [reply modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm]: I don't know who coded that lol
I guess technically it still works but I can fix it

**Comment by softcerv** [comment]
This is a PR that affects the balance of cargo by adding items and modifying prices, this should be marked as a balance PR rather than QoL.

**Review by GoldenAlpharex — APPROVED** [review]
This is fine with me, but I don't really think this counts as a QoL, sorry.

**Pull request merged**